packages/admin/blocks-admin/src/blocks/factories/columnsBlock/FinalFormLayoutDisplay.tsx

```diff
@@ -0,0 +1,35 @@
+import { ListItemText } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { ColumnsBlockLayout } from "../createColumnsBlock";
+
+interface Props extends FieldRenderProps<ColumnsBlockLayout> {
+    layouts: ColumnsBlockLayout[];
+}
+
+export function FinalFormLayoutDisplay({ layouts }: Props): React.ReactElement {
```

Comment:
Maybe you should check if `layouts` is an empty array and either return `null` or show some different text. 
Otherwise this component could crash. 

```suggestion
export function FinalFormLayoutDisplay({ layouts }: Props) {
    if (layouts.length === 0) return null;

```